src/components/content/ContentComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import DetailBlogComponent from '../detail_blog/DetailBlogComponent';
+import ListBlogComponent from '../list-blog/ListBlogComponent';
+import './ContentStyle.css';
+
+const topicsBlog = [
+    {
+        name: 'Front-end',
+    },
+    {
+        name: 'Back-end',
+    },
+    {
+        name: 'Fullstack',
+    },
+
+]
+function ContentComponent() {
+    return (
+        <div>
+            <div className="row">
+                <div className="leftcolumn">
+                    <ListBlogComponent />
+                </div>
+                <div className="rightcolumn">
+                    <div className="card">
+                        {
+                            topicsBlog.map((item, key) => (
+                                <button key = {key}className={`bg-transparent hover:bg-cyan-500 text-blue-700 font-semibold hover:text-white py-2 px-4 border border-blue-500 hover:border-transparent rounded`}>
```

Comment:
Don't use the index as a key, you should make an Id for an item in your arr and use this. Also, the className is a string so you don't need to use `{}` to wrap it.
I saw that you are using both tailwind and normal CSS. I think you should combine it by using SCSS and apply the style of tailwind to it or you can use any CSS in JS like styled-components with tailwind.